release-cloudbuild-preparation.yaml

```diff
@@ -0,0 +1,153 @@
+steps:
+  # login to Dockerhub
+  - name: gcr.io/cloud-builders/docker
+    args:
+      - '-c'
+      - docker login --username=$$USERNAME --password=$$PASSWORD
+    id: docker-login
+    entrypoint: bash
+    secretEnv:
+      - USERNAME
+      - PASSWORD
+
+  # Build base image
+  - name: gcr.io/cloud-builders/docker
+    args:
+      - '-c'
+      - |
+        docker build . \
+          --platform linux/amd64 \
+          -f docker/zenml-dev.Dockerfile \
+          -t $$USERNAME/prepare-release:base
```

Comment:
I think for all of these, it would be best if they included the version in the tag as well, e.g. `prepare-release:base-0.68.0`. This way, we don't accidentally end up using an image that we don't want and run into some weird behaviour.

Reply:
I will make the adjustments.

Reply:
Fixed this one as well.